internal/events/types/extension_transaction/transaction_event.go

```diff
@@ -31,12 +34,55 @@ func (e Event) GenerateEvent(params events.MockEventParameters) (events.MockEven
 	var event []byte
 	var err error
 
+	clientID := viper.GetString("clientId")
+
+	// if not configured, generate a random one
+	if clientID == "" {
+		clientID = util.RandomClientID()
+	}
+
 	if params.Cost == 0 {
 		params.Cost = 100
 	}
 	switch params.Transport {
 	case models.TransportEventSub:
-		return events.MockEventResponse{}, errors.New("Extension transactions are unsupported on Eventsub")
+		body := &models.TransactionEventSubResponse{
+			Subscription: models.EventsubSubscription{
+				ID:      params.ID,
+				Status:  "enabled",
+				Type:    triggerMapping[params.Transport][params.Trigger],
+				Version: "1",
+				Condition: models.EventsubCondition{
+					ExtensionClientID: clientID,
+				},
+				Transport: models.EventsubTransport{
+					Method:   "webhook",
+					Callback: "null",
+				},
+				Cost:      1,
+				CreatedAt: util.GetTimestamp().Format(time.RFC3339Nano),
+			},
+			Event: models.TransactionEventSubEvent{
+				ID:                   params.ID,
+				ExtensionClientID:    clientID,
+				BroadcasterUserID:    params.ToUserID,
+				BroadcasterUserLogin: "testBroadcaster",
+				BroadcasterUserName:  "testBroadcaster",
+				UserName:             "testUser",
+				UserLogin:            "testUser",
+				UserID:               params.FromUserID,
+				Product: models.TransactionEventSubProduct{
+					Name:          "Test Trigger Item from CLI",
```

Comment:
Hi, would it be possible that you can do the following,
- make the TransactionEventSubProduct.Name field use params.ItemName
- make the TransactionEventSubProduct.SKU field use params.ItemId

or default to the values currently in use!

this will make it so much easier to test my bits functionality in my backend

Reply:
I'll get these changes in after I merge in; I have a bunch of work pending on the CLI that'll be in the upcoming release. Going to merge now.